src/org/intellij/erlang/inspection/ErlangUndefinedCallbackFunctionInspection.java

```diff
@@ -64,26 +65,30 @@ protected void checkFile(@NotNull ErlangFile file, @NotNull ProblemsHolder probl
         builder.append("'").append(ErlangPsiImplUtil.createFunctionPresentationFromCallbackSpec(spec)).append("', ");
       }
       String message = builder.substring(0, builder.length() - 2);
-      registerProblem(problemsHolder, behaviourRef, message, new MyLocalQuickFixBase(undefinedCallbacks));
+      registerProblem(problemsHolder, behaviourRef, message, new CreateAndExportFunctionsFix(undefinedCallbacks));
     }
   }
 
-  private static class MyLocalQuickFixBase extends LocalQuickFixBase {
+  private static class CreateAndExportFunctionsFix extends LocalQuickFixBase {
     @NotNull
-    private final Collection<ErlangCallbackSpec> myCallbackSpecs;
+    private final Collection<SmartPsiElementPointer<ErlangCallbackSpec>> myCallbackSpecs;
```

Comment:
Please, avoid using Collection. You iterate over it, expecting some concrete iteration order. Collection doesn't bring you this property. Why not use list instead?